### PR TITLE
Fix dev and a logging message.

### PR DESCRIPTION
**Pull request by marc-barry**
@Shopify/edgescale 

This pull request does a few things:
 - Update Go
 - Create a `railgun.yml` file (see https://engineering.shopify.io/tools/dev/railgun/Railgun-Config)
 - Adjust a logging message which was missing `gozk-recipes/session`

```
marc:gozk-recipes marcbarry$ dev up
⭑ 1/4 Prerequisites
⭑ 2/4 Homebrew Packages
⭑ 3/4 Golang
┏━━ 👩‍💻  4/4 Railgun ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
┃ ⭑ install railgun 2.3.2
┃ ⭑ add project to Railgun.app
┃ ⭑ download railgun image (railgun-os-stage2-0.1.10.sfs)
┃ 𝒾 start railgun
┃ ✓ (docker)
┃ ✓ (openssh)
┃ ✓ toxiproxy
┃ ✓ zookeeper
┃ ✓ start railgun: done
┃ ⭑ add hosts to /etc/hosts
┃ ⭑ launch railgun app
┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ (8.93s) ━━
```
```
marc:gozk-recipes marcbarry$ dev test
👩‍💻  Running script/test from dev.yml
=== RUN   TestCreateAndMaintain
waiting 10.5 seconds for zookeeper to purge an ephemeral node...
re-establishing proxy
waiting 20 seconds for maintainEphemeral to time out waiting for connection re-establishment
--- PASS: TestCreateAndMaintain (31.06s)
PASS
ok  	github.com/Shopify/gozk-recipes/ephemeral	31.074s
?   	github.com/Shopify/gozk-recipes/examples	[no test files]
?   	github.com/Shopify/gozk-recipes/lock	[no test files]
=== RUN   TestChildrenRecursiveWithNonExistingNodeShouldHaveEmptyResult
--- PASS: TestChildrenRecursiveWithNonExistingNodeShouldHaveEmptyResult (0.01s)
=== RUN   TestChildrenRecursiveWithNodesShouldHaveResult
--- PASS: TestChildrenRecursiveWithNodesShouldHaveResult (0.02s)
=== RUN   TestChildrenRecursiveWithLimitedDepthShouldReturnSubset
--- PASS: TestChildrenRecursiveWithLimitedDepthShouldReturnSubset (0.01s)
=== RUN   TestCreateRecursiveAndSetWithNoParentsShouldCreateNodes
--- PASS: TestCreateRecursiveAndSetWithNoParentsShouldCreateNodes (0.02s)
=== RUN   TestCreateRecursiveAndSetWithParentsShouldNotChangeData
--- PASS: TestCreateRecursiveAndSetWithParentsShouldNotChangeData (0.02s)
=== RUN   TestDeleteRecursiveShouldDelete
--- PASS: TestDeleteRecursiveShouldDelete (0.01s)
=== RUN   TestReceiveEventWhenSubscribing
--- PASS: TestReceiveEventWhenSubscribing (1.34s)
=== RUN   TestResumeZKSessionWithValidSession
--- PASS: TestResumeZKSessionWithValidSession (0.01s)
=== RUN   TestResumeZKSessionFailsWithInvalidClientId
--- PASS: TestResumeZKSessionFailsWithInvalidClientId (0.00s)
=== RUN   TestResumeZKSessionWithInvalidClientIdDoesNotDisconnectExistingSession
--- PASS: TestResumeZKSessionWithInvalidClientIdDoesNotDisconnectExistingSession (0.00s)
	session_test.go:151: Existing session was not disconnected by ResumeZKSession with invalid clientId
PASS
ok  	github.com/Shopify/gozk-recipes/session	1.462s
?   	github.com/Shopify/gozk-recipes/test	[no test files]
```

### CODE REVIEW

**Review by sbfaulkner — APPROVED** [review]

File: railgun.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+volumes:
+  root:  '2G'
+
+services:
+  - toxiproxy
+  - zookeeper
+
+hostnames:
+  - gozk-recipes.myshopify.io
```
Comment (sbfaulkner) [inline railgun.yml]: NL ?

Comment (marc-barry) [reply railgun.yml]: Added.

File: dev.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ name: gozk-recipes
 up:
   - homebrew:
     - zookeeper
-  - go: 1.7.5
+  - go: 1.9.2
```
Comment (charlescng) [inline dev.yml:6]: How are we building the production binary?

I have noticed that we use godeb and it's completely broken.

We could potentially write code that will not compile in production.

Comment (marc-barry) [reply dev.yml:6]: This is a library and doesn't produce a binary. We need Go for running the tests.

File: railgun.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# https://dev-accel.shopify.io/dev/railgun/Railgun-Config
+name: gozk-recipes
+
+vm:
+  image:      /opt/dev/misc/railgun-images/default
+  ip_address: 192.168.64.87
+  memory: 2G
```
Comment (jpittis) [inline railgun.yml]: Do we really need 2G memory? (Not to say this really matters)

Comment (jpittis) [reply railgun.yml]: I guess this is legacy. Whatever :)

**Review by jpittis — APPROVED** [review]
🐑

**Review by charlescng — APPROVED** [review]

File: railgun.yml (hunk 3 above)
Comment (marc-barry) [reply railgun.yml]: Trying with something smaller.

Comment (marc-barry) [reply railgun.yml]: I tried some different values the minimum I could use and have the services start was `200M` and so I'm running with that.

**Pull request merged**